app/experimenter/experiments/forms.py

```diff
@@ -153,6 +153,26 @@ class Meta:
             "proposed_enrollment",
         ]
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # The Experiment.owner field was added late. So it's still nullable
+        # in the model.
+        # Because of that, we can't set `required=True` on the `owner` field
+        # and we can't remove the "empty label" (e.g
+        # <option value="">-------------</option>) option. What if you're
+        # editing an experiment that was created *before* the `owner` field
+        # was prefilled?
```

Comment:
@peterbe I appreciate the forethought here but I don't think this is a case we need to handle, I think adding required=True to the form field is the correct solution here.

Reply:
But if you try to edit an experiment that doesn't have an owner it would have to set the owner to the best first (alphabetically sorted) in the `<select>` to become the owner. 

If you undo my more complex solution I think we should write a data migration to set an owner for all existing experiments that did NOT have an owner. But to which user???

Reply:
@peterbe It's not an invalid concern but no existing studies have an empty owner, and you can't edit past studies so the case of trying to edit an experiment where owner is blank isn't a case users can access.

Reply:
/me simplified the code